src/static/css/2019.css

```diff
@@ -333,7 +333,6 @@ footer .language-switcher {
 }
 footer .copyright {
   grid-area: copyright;
-  display: flex;
```

Comment:
AFAICT this changes the alignment of the copyright with the social links. I think the correct fix is to ensure there's a space (not a newline) between "under" and the link in the template.

One other thing I'd ask you to fix is to ensure the color of the link is white by default and yellow on hover, like other footer links.

Reply:
> AFAICT this changes the alignment of the copyright with the social links.

I don't see it. Can you explain more what you mean?

> I think the correct fix is to ensure there's a space (not a newline) between "under" and the link in the template.

That new line is not there in the source code:

![image](https://user-images.githubusercontent.com/10931297/69584397-01bf8200-0fd5-11ea-932b-de2bafb91367.png)

Chrome dev tools is adding that automatically, if you saw it in Element inspector.

BTW it also works adding a `&nbsp` instead of the space. Basically seems the browser is treating these two items as separate items where flat space between them can be collapsed because of the `flex` property.

> One other thing I'd ask you to fix is to ensure the color of the link is white by default and yellow on hover, like other footer links.

That's a fair ask - added now!

What do you think about the underline? I think keep it even though other nav items on home page don't have it cause otherwise not obvious it's a link without it.

Also we have "Copyright ©" and I think it's redundant to have both the text and the symbol. It also makes the line stretch to two lines on most mobiles which looks ugly:

![image](https://user-images.githubusercontent.com/10931297/69583942-e99b3300-0fd3-11ea-924c-a3fe34f26238.png)

Let me know if you're cool removing the text and just using the © symbol (on both Desktop and Mobile):

![image](https://user-images.githubusercontent.com/10931297/69583845-b193f000-0fd3-11ea-9ef2-1d2a1966fc4c.png)

Reply:
It's vertically aligned to the top:

![image](https://user-images.githubusercontent.com/1120896/69585284-3fa8b000-0fad-11ea-880d-0b58dc2dda54.png)

Originally it was vertically centered:

![image](https://user-images.githubusercontent.com/1120896/69585299-49caae80-0fad-11ea-8363-39d36167680c.png)

Reply:
OK went with [a different fix](https://stackoverflow.com/questions/44432447/displayflex-with-text-and-tag-inside-removes-spaces) in the [latest commit](https://github.com/HTTPArchive/almanac.httparchive.org/pull/543/commits/0cf3811aef5369e94ca6d4fa97f51693ee3831db). Think that is a bit more a hack but seems to work fine.

Reply:
Actually I had to do the `<span>` thing as otherwise it still put the link to the right on mobile:

![image](https://user-images.githubusercontent.com/10931297/69576961-46432180-0fc5-11ea-85ec-55f815e61d65.png)